CHANGELOG.md

```diff
@@ -4,9 +4,10 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased
 
+- SUP-805: SUP-805: Team resource API retries [[PR #360](https://github.com/buildkite/terraform-provider-buildkite/pull/360)] @james2791
 - SUP-1392: Random test suite names in tests/t.Run() conversion [[PR #376](https://github.com/buildkite/terraform-provider-buildkite/pull/376)] @james2791
 - SUP-1374 Add timeout to provider and cluster datasource [[PR #363](https://github.com/buildkite/terraform-provider-buildkite/pull/363)] @jradtilbrook
-    - SUP-1374 Remove timeout context [[PR #378](https://github.com/buildkite/terraform-provider-buildkite/pull/378)] @jradtilbrook
```

Comment:
Looks like a bit of a bad merge removing this line

Reply:
A recent main merge did it earlier today, resolved it below on line 10 ;)